diff --git a/bert.cpp b/bert.cpp
index 74fcc03..66a1182 100644
--- a/bert.cpp
+++ b/bert.cpp
@@ -685,7 +685,7 @@ struct bert_ctx * bert_load_from_file(const char *fname)
 
         // TODO: Max tokens should be a param?
         int32_t N = new_bert->model.hparams.n_max_tokens;
-        new_bert->mem_per_input = 1.1 * (new_bert->mem_per_token * N); // add 10% to account for ggml object overhead
+        new_bert->mem_per_input = 1.5 * (new_bert->mem_per_token * N); // add 50% to account for ggml object overhead
 
     }
     printf("%s: mem_per_token %d KB, mem_per_input %lld MB\n", __func__, new_bert->mem_per_token / (1 << 10), new_bert->mem_per_input / (1 << 20));
